From fad5fcafeebf08cc43f56997a8c945e5aef25c92 Mon Sep 17 00:00:00 2001
From: Shubham Bhise <shubhamprince111@gmail.com>
Date: Fri, 26 Nov 2021 14:10:18 +0200
Subject: [PATCH] bionic: Make libstdc++ as product_available

 * Required for libjni_latinimegoogle
   Build error : https://www.toptal.com/developers/hastebin/iruvelewuf.less

Change-Id: I7b1d9062e687457208f6fbe9692931693dd8a2f4
(cherry picked from commit 3a1df1c894b372425286cbf379ef3273cc1f2583)
---
 libc/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libc/Android.bp b/libc/Android.bp
index 98b878151..cb07f5f60 100644
--- a/libc/Android.bp
+++ b/libc/Android.bp
@@ -2021,6 +2021,7 @@ cc_library {
     static_ndk_lib: true,
     static_libs: ["libasync_safe"],
     vendor_available: true,
+    product_available: true,
 
     static: {
         system_shared_libs: [],
